doc/lsst.daf.butler/datastores.rst

```diff
@@ -9,7 +9,7 @@ Datastore Configuration
 A Butler `Datastore` is configured in the ``datastore`` section of the top-level Butler YAML configuration.
 The only mandatory entry in the datastore configuration is the ``cls`` key.
 This specifies the fully qualified class name of the Python class implementing the datastore.
-The default Butler configurations uses the `~datastores.posixDatastore.PosixDatastore`.
+The default Butler configurations uses the `~datastores.fileDatastore.FileDatastore`.
```

Comment:
"configurations uses" should be either "configurations use" or "configuration uses".